### ISSUE TITLE
Configmap creation from .yaml

### ISSUE THREAD

**Issue by jbsiii** [rancher/rancher#16610]
*Describe your issue here*
Rancher offers the ability to create Configmaps by entering name/value pairs via the UI.  The multi-add feature is just OK.  It would be better if it also added the ability to create by uploading a file of name-value pairs.
Also, the UI offers the ability to download the .yaml file for a configmap, but no ability to create a new configmap by uploading a .yaml file.  This makes it difficult to create these resources via script out of a source controlled system.

---
| Useful | Info |
| :-- | :-- |
|Versions|Rancher `v2.1.1` UI: `v2.1.4` |
|Route|`authenticated.cluster.index`|

**Comment by loganhz** [comment]
User will be able to upload a file contains key/value pairs which splitted by `=` or `: ` in 2.2
For example,

```
a=b
c=d
```
And it will some `Import YAML` button in config map list page

**Comment by davidnuzik** [comment]
Version: master 2.2 (1/4/19)
Confirmed feature. I tested the Import YAML option with a file, as well as manually entering in UI and also tested reading from file via the "Add Config Map" screen and ensured copy/paste is still working correctly. Halo tested, tried to break things, test with unicode and unique chars, etc. Validate HTTP and API.
It is important to have your text file have a newline after each key=value. key:pair does not work and I will enter an issue for this if necessary.
when using YAML it must of course be appropriately structured. I tested with manually plugging in data: in the yaml
